### ISSUE TITLE
Remove Noise paremter injecto in Vuvuzela segment (Trac #2243)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2235]
As per Andrii Terliuk's presentation in the IceCube bi-weekly call (22 January 2019):

[https://drive.google.com/drive/folders/1DAiEVsRibOBXlRKsWO6lelsIdWmG29K4]

Migrated from https://code.icecube.wisc.edu/ticket/2243
```json
{
    "status": "closed", 
    "changetime": "2019-02-27T20:09:10", 
    "description": "As per Andrii Terliuk's presentation in the IceCube bi-weekly call (22 January 2019):\n\n[https://drive.google.com/drive/folders/1DAiEVsRibOBXlRKsWO6lelsIdWmG29K4]", 
    "reporter": "juancarlos", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1551298150937247", 
    "component": "combo simulation", 
    "summary": "Remove Noise paremter injecto in Vuvuzela segment", 
    "priority": "major", 
    "keywords": "Vuvuzela", 
    "time": "2019-02-27T20:08:12", 
    "milestone": "", 
    "owner": "juancarlos", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20190227T20:09:03: juancarlos commented:

> Parameters should be read from GCD file.

**Comment by icecube-trac** [comment]
Trac update at 20190227T20:09:10: 
* juancarlos commented:

> In changeset 171602:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="171602"
> Closes #2235. Parameters should be read from GCD file.
> ```


* juancarlos changed status from "new" to "closed"
* juancarlos changed resolution from "" to "fixed"
